src/core/process/round.test.ts

```
@@ -10,7 +10,25 @@ import { SpanStatusCode } from "@opentelemetry/api";
+    prepareRouter,
+    finalizeRound,
+    initializeRound,
+    processOrderInit,
+} from "./round";
+
+vi.mock("sushi/currency", async (importOriginal) => {
+    return {
+        ...(await importOriginal()),
+        Token: class {
+            constructor(args: any) {
+                return { ...args };
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)